frontend/packages/ceph-storage-plugin/src/components/block-pool/body.tsx

```diff
@@ -64,6 +65,23 @@ export const BlockPoolBody = (props: BlockPoolBodyPros) => {
   const [storageCluster, storageClusterLoaded, storageClusterLoadError] = useK8sGet<
     ListKind<StorageClusterKind>
   >(OCSServiceModel, null, CEPH_STORAGE_NAMESPACE);
+  const [pools, poolsLoaded, poolsLoadedError] = useK8sGet<ListKind<StoragePoolKind>>(
+    CephBlockPoolModel,
+    null,
+    CEPH_STORAGE_NAMESPACE,
+  );
+
+  React.useMemo(() => {
+    if (!state.failureDomain && poolsLoaded && !poolsLoadedError) {
+      const pool: StoragePoolKind = pools?.items.find((blockPoolConfig) =>
+        isDefaultPool(blockPoolConfig),
+      );
```

Comment:
```suggestion
      const pool: StoragePoolKind = pools?.items.find(isDefaultPool);
```

Reply:
Ah! you told this in earlier PR comment, I forgot, my bad :)